CHANGELOG.md

```diff
@@ -28,6 +28,9 @@
   - `GP_GetCurrentSetlistIndex()` (C) and `GigPerformerFunctions::getCurrentSetlistIndex()` (C++)
   - `GP_SwitchToSetlist()` (C) and `GigPerformerFunctions::switchToSetlist(int setlistIndex)` (C++)
 
+- Changed .gitignore:
+  - Added .vs/ to rpevent Visual Studio IDEs from littering the project
+
```

Comment:
Thanks for making the effort to include this in the changelog. However, this is not relevant to extension developers (the target audience of this file), so I am going to remove these lines.